### PR TITLE
kafka v2 hotfixes

### PR DESCRIPTION
**Pull request by jochman**
added docker
removed 'required' args from fetch incidents
removed `enable debug` title
## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation https://github.com/demisto/etc/issues/15427
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Kafka_V2.yml [hunk 1]
```diff
@@ -435,7 +438,7 @@ script:
 
         if demisto.command() == 'test-module':
             # This is the call made when pressing the integration test button.
-            test_module()
+            test_module(
```
Comment (Itay4) [inline Integrations/integration-Kafka_V2.yml]: you are missing `)`

File: Integrations/integration-Kafka_V2.yml [hunk 2]
```diff
@@ -350,7 +350,7 @@ script:
             return_error('Topic {} was not found in Kafka'.format(topic))
 
 
-    def fetch_partitions():
+      def fetch_partitions():
```
Comment (Itay4) [inline Integrations/integration-Kafka_V2.yml]: remove whitespace

File: Integrations/integration-Kafka_V2.yml [hunk 3]
```diff
@@ -438,7 +438,7 @@ script:
 
         if demisto.command() == 'test-module':
             # This is the call made when pressing the integration test button.
-            test_module()
+            test_module(
```
Comment (yaakovi) [inline Integrations/integration-Kafka_V2.yml]: return the bracket

File: Integrations/integration-Kafka_V2.yml [hunk 4]
```diff
@@ -353,7 +350,7 @@ script:
             return_error('Topic {} was not found in Kafka'.format(topic))
 
 
-    def fetch_partitions():
+      def fetch_partitions():
```
Comment (yaakovi) [inline Integrations/integration-Kafka_V2.yml]: fix the indentation

File: Integrations/integration-Kafka_V2.yml [hunk 5]
```diff
@@ -95,14 +96,19 @@ script:
     BROKERS = demisto.params().get('brokers')
 
     # Should we use SSL
-    USE_SSL = demisto.params().get('insecure', False)
+    USE_SSL = not demisto.params().get('insecure')
```
Comment (Itay4) [inline Integrations/integration-Kafka_V2.yml]: need to have default: `VERIFY_SSL = not demisto.params().get('insecure', False)`

File: Integrations/integration-Kafka_V2.yml [hunk 6]
```diff
@@ -7,18 +7,19 @@ category: Messaging
...
   name: brokers
   defaultvalue: ""
   type: 0
   required: true
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
 - display: Do not validate server certificate (insecure)
```
Comment (Itay4) [inline Integrations/integration-Kafka_V2.yml]: change according to our standard: `Trust any certificate (unsecure)`

File: Integrations/integration-Kafka_V2.yml [hunk 7]
```diff
@@ -7,18 +7,19 @@ category: Messaging
...
 description: The Open source distributed streaming platform
 configuration:
-- display: Use proxy
-  name: proxy
-  defaultvalue: ""
-  type: 8
-  required: false
 - display: 'CSV list of Kafka brokers to connect to, e.g., 172.16.20.207:9092,172.16.20.234:9093'
```
Comment (Itay4) [inline Integrations/integration-Kafka_V2.yml]: remove comma after `e.g.`

**Pull request merged**